### PR TITLE
compute angle between two vectors

### PR DESCRIPTION
**Pull request by geemi725**

### CODE REVIEW

File: data/angle/angle.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+v2_u = unit_vec(v2)
+
+angle = np.arccos(np.clip(np.dot(v1_u, v2_u), -1.0, 1.0))
+
+angle_codex =  vector_angle(v1,v2)
+
+if np.isclose(angle, angle_codex) == True:
+    result = True
+else:
+    result = False
```
Comment (hgandhi2411) [inline data/angle/angle.py:19]: This is logically correct but can be simplified into one line like in other PRs: `result = True if np.isclose(angle, vector_angle(v1,v2)) else False` Up to you if you'd like to change it.

File: data/angle/angle.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: angle
```
Comment (hgandhi2411) [inline data/angle/angle.yml:1]: Your function name is vector_angle

**Review by hgandhi2411 — CHANGES_REQUESTED** [review]
Looks good, commented for some minor changes.

File: data/angle/angle.yml (hunk 2 above)
Comment (navneeth3005) [reply data/angle/angle.yml:1]: I dont think this is a problem.
The function name can be different from the file name. 
The file name should appear in the yml file. 
See cubes.

**Review by navneeth3005 — APPROVED** [review]

File: data/angle/angle.yml (hunk 2 above)
Comment (hgandhi2411) [reply data/angle/angle.yml:1]: Thanks for clarifying @navneeth3005

**Review by hgandhi2411 — APPROVED** [review]

**Pull request merged**